src/img/icons/caret-right-small.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="6" height="10" viewBox="0 0 6 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0)">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M0.469828 9.88039C0.569828 9.98039 0.679828 10.0304 0.729828 9.98039L5.51983 5.20038C5.53674 5.18347 5.53935 5.15225 5.52917 5.11429C5.53509 5.07969 5.52923 5.04979 5.50985 5.03041L5.42789 4.94862C5.42525 4.94586 5.42256 4.94312 5.41983 4.94038L5.41226 4.93303L0.719846 0.250411C0.679846 0.21041 0.559846 0.250411 0.459846 0.350411L0.119846 0.700409C0.0198461 0.80041 -0.0301541 0.91041 0.0198456 0.96041L4.17917 5.11105L0.0198278 9.27039C-0.0201722 9.31039 0.0198283 9.43039 0.119828 9.53039L0.469828 9.88039Z" fill="#3C444D" fill-opacity="0.7"/>
+</g>
+<defs>
+<clipPath id="clip0">
+<rect width="5.53" height="9.77" fill="white" transform="translate(5.52979 10) rotate(180)"/>
+</clipPath>
+</defs>
+</svg>
```

Comment:
This svg needs to be updated @rdig  maybe you could do call with full dapp team about better svg optimisation ?

Reply:
I think it's a great idea. I would appreciate that.